pacemaker.spec.in

```diff
@@ -355,6 +355,10 @@ install -m 644 mcp/pacemaker.combined.upstart ${RPM_BUILD_ROOT}%{_sysconfdir}/in
 install -m 644 tools/crm_mon.upstart ${RPM_BUILD_ROOT}%{_sysconfdir}/init/crm_mon.conf
 %endif
 
+%if %{defined _unitdir}
+mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/lib/rpm-state/%{name}
+%endif
```

Comment:
That suggestion may have something to do with http://rpm.org/gitweb?p=rpm.git;a=commitdiff;h=c7523b4c, also note this former brokeness: https://bugzilla.redhat.com/839656.
I am now inclined to think it'd be sane to mark the directory also `%dir` for good measure.